src/Task/PhpCsFixer.php

```diff
@@ -102,7 +102,7 @@ public function run(ContextInterface $context): TaskResultInterface
 
         if (!$process->isSuccessful()) {
             $messages = [$this->formatter->format($process)];
-            $fixerCommand = $this->formatter->formatSuggestion($process);
+            $fixerCommand = $process->getCommandLine();
```

Comment:
Instead of creating the suggestion in the formatter, we can add arguments to the `$arguments` variable as we do in the eslint task:
https://github.com/phpro/grumphp/blob/master/src/Task/ESLint.php#L92-L105

This way, we are still suggesting the fix command to the end user in case of a detected fixable item.

Reply:
Fixed, thanks!